arex-instrumentation/database/arex-database-mybatis3/src/main/java/io/arex/inst/database/mybatis3/ExecutorInstrumentation.java

```diff
@@ -102,7 +105,7 @@ public static void onExit(@Advice.Argument(0) MappedStatement var1,
             }
 
             if (ContextManager.needRecord()) {
-                InternalExecutor.record(var1, var2, boundSql, result, throwable, METHOD_NAME_QUERY);
+                InternalExecutor.record(var1, var2, boundSql, result, throwable, METHOD_NAME_QUERY, originalSql);
```

Comment:
Suggest directly replace `boundSql` with `originalSql`

Reply:
got it